spec/lib/pwwka/error_handlers/chain_spec.rb

```diff
@@ -0,0 +1,10 @@
+require "spec_helper"
+
+describe Pwwka::ErrorHandlers::Chain do
+  let(:subject) { described_class.new(default_handler_chain) }
+
+  describe "#handle_error" do
+    xit "logs exceptions that occur in the error handling chain" do
```

Comment:
is this WIP? if not please remove.

Reply:
Yes, this is all WIP. Before coding up my thoughts, I wanted to make sure there wasn't more context I was missing.

Reply:
👍